### DISCUSSION TITLE
Do we have a benchmark to compare with other json lib?

### DISCUSSION THREAD

**Discussion by echobinarybytes** [nlohmann/json#4244] [Ideas]
Need a benchmark

**Comment by kevinbcarpenter** [comment]
This benchmark includes this library as well as others.

https://github.com/miloyip/nativejson-benchmark
[https://opengraph.githubassets.com/3d7642963d7a4f17ca1a88408ff6102b977c032344f82caa0fbf72f27bbc0618/miloyip/nativejson-benchmark]<https://github.com/miloyip/nativejson-benchmark>
GitHub - miloyip/nativejson-benchmark: C/C++ JSON parser/generator benchmark<https://github.com/miloyip/nativejson-benchmark>
C/C++ JSON parser/generator benchmark. Contribute to miloyip/nativejson-benchmark development by creating an account on GitHub.
github.com

Kevin

________________________________
From: uubs ***@***.***>
Sent: Thursday, December 14, 2023 10:56 PM
To: nlohmann/json ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [nlohmann/json] Do we have a benchmark to compare with other json lib? (Discussion #4244)


Need a benchmark

**Reply by nlohmann** [reply]
It's very outdated, and since libraries faster than RapidJSON were published, the author seem to have lost interest in updating it ;)
